paasta_tools/chronos_tools.py

```diff
@@ -477,8 +473,8 @@ def create_complete_config(service, job_name, soa_dir=DEFAULT_SOA_DIR):
     # this makes it trivial to compare configs and know when to bounce.
     complete_config['description'] = get_config_hash(complete_config)
 
-    # If the job was previously stopped, we should stop the new job as well
-    # NOTE this clobbers the 'disabled' param specified in the config file!
+    # use the 'description' field to determine the disabled state
+    # of the job
     if desired_state == 'start':
```

Comment:
This is.. not quiet right now.

The description field on line 474 needs to be calculated after this desired state. (after 481)

I guess this is a different bug.